src/RequestObject/RequestObjectFactory.php

```diff
@@ -55,11 +53,7 @@ public function __construct(
     ) {
         $this->algorithmManager = $algorithmManager ?? (new AlgorithmManagerBuilder())->build();
         $this->jwsBuilder = $jwsBuilder ?? new JWSBuilder($this->algorithmManager);
-        $this->jweBuilder = $jweBuilder ?? new JWEBuilder(
-            $this->algorithmManager,
-            $this->algorithmManager,
-            new CompressionMethodManager([new Deflate()])
-        );
+        $this->jweBuilder = $jweBuilder ?? new JWEBuilder($this->algorithmManager);
```

Comment:
https://web-token.spomky-labs.com/migration/from-v1.x-to-v2.0-2#simplified-algorithm-manager